demos/tests/image_sequences.py

```diff
@@ -136,15 +148,71 @@
         image_net_arg('00048316'),
     ],
 
-    'smart-classroom-demo': [
-        image_net_arg('00000074'),
-        image_net_arg('00000141'),
-        image_net_arg('00000141'),
-        image_net_arg('00000164'),
-        image_net_arg('00000181'),
-        image_net_arg('00000164'),
-        image_net_arg('00000181'),
+    'py/action-recognition': [
```

Comment:
Agree, but some image sets are used only for a single python demo and I thought it would be easier to read. Do you insist on removing `py/` prefix?

Reply:
> Agree, but some image sets are used only for a single python demo and I thought it would be easier to read.

I wouldn't count out the possibility of a similar native demo appearing later.

> Do you insist on removing py/ prefix?

Yes, unless you can convince me that you shouldn't. :-)

Reply:
renamed

Reply:
Please rearrange these sequences to keep the names in alphabetical order, as well.

Reply:
rearranged